packages/env/lib/init-config.js

```diff
@@ -203,6 +203,14 @@ function wordpressDockerFileContents( image, config ) {
 
 	return `FROM ${ image }
 
+# Update apt sources for archived versions of Debian.
```

Comment:
I might also add a comment linking to that stack overflow thread for a bit of extra context

Reply:
This isn't a bad idea. Given that it's a better source of truth, I've instead linked to [the Debian mailing list record of the change](https://lists.debian.org/debian-devel-announce/2023/03/msg00006.html).